docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,199 @@ which groups these values were observed, as well as the difference and ratio
+them as:
+
+* *The Solutionism Trap*
+
+* *The Ripple Effect Trap*
+
+* *The Formalism Trap*
+
+* *The Portability Trap*
+
+* *The Framing Trap*
+
+We provide some definitions and examples of these traps to help Fairlearn
+users think about how choices they make in their work can avoid these common pitfalls.
+
+The Solutionism Trap
+^^^^^^^^^^^^^^^^^^^^
+
+This trap occurs when we assume that the best solution to a problem
+may involve technology, and fail to recognize other possible solutions outside of
+this realm. This trap also occurs as definitions of fairness are
+contested and changed and models become obsolete.
+
+Example: consider the problem of internet connectivity in rural communities.
+An example of the solutionism trap is assuming that by using data science to
+study internet speed in a given region, policies and deals can be arranged to
+reduce the price of internet access. However, if there are additional socioeconomic
+problems within a community, for example lack of education, infrastructure, information
+technology and health services, then an algorithmic solution purely focused on internet
+speed will fail to meaningfully address the needs of the community.
+
+As an example, the Federation of Canadian municipalities released a report
+ tackling both connectivity and infrastructure needs of rural communities to
+more meaningfully address this social issue
+https://fcm.ca/sites/default/files/documents/resources/report/broadband-access-rural-canada.pdf.
+
+The Ripple Effect Trap
+^^^^^^^^^^^^^^^^^^^^^^
+
+This trap occurs when we do not study the unintended consequences of introducing
+technology into an existing social system. Such consequences include change in
+behaviour, outcomes individual experience and a change in underlying social values
+and incentives of a given social system, for instance by increasing perceived value
+of quantifiable metrics over non-quantifiable ones.
+
+Example: consider the problem of banks deciding whether an individual should
+be approved for a loan. Prior to using machine learning algorithms
+to compute a "score", banks might rely on engaging in conversations with
+clients, recommending a plan based on their unique situation, and
+discussing with other team members to obtain feedback. By introducing an
+algorithm, it is possible that banks stop engaging in conversations
+with team members and clients, and assume the algorithm is good enough
+to blindly accept a recommendation. Furthermore, some might use
+information on people algorithm typically rejects to create a system
+that takes advantage of people who need funds but did not receive them.
+
+To avoid this pitfall, we must be aware that once a technology is incorporated
+into a social context, new groups may reinterpret it and potentially use this
+information for outside purposes which include nefarious ones. To
+solve for this issue, we must incorporate "what if" scenarios into testing
```

Comment:
```suggestion
help address this issue, we should adopt "what if" scenarios to envision how the social context might change after introducing a model, including how it may change the power dynamics of existing groups in that context.
```